ci_framework/roles/run_hook/README.md

```diff
@@ -24,7 +24,7 @@ In such a case, the following data can be provided to the hook:
 
 #### Hook callback
 A hook may generate new parameters that will be fed into the main play. In order
-to do so, the hook playbook has to create a YAML file as follow:
+to do so, the hook playbook has to create a YAML file as follows:
```

Comment:
uho, I didn't know about that one. "follows", really? I'll try to remember that.